### DISCUSSION TITLE
alpine 3.21 repo?

### DISCUSSION THREAD

**Discussion by madsi1m** [nginx/nginx#375] [Q&A]
Hi, I noticed there is no mainline alpine repo for alpine 3.21.
https://nginx.org/packages/mainline/alpine/v3.21/main/x86_64/

Any plans to support the newer alpine?
Is it ok to point my alpine to an older version of the repo like v3.20?

**Comment by thresheek** [comment]
Hi @madsi1m - it is indeed planned.

As we use AWS EC2 to build our packages, we waited for Alpine team to add AMIs on AWS for that new release - which had just happened a couple of hours ago.  We would need to run the tests & builds to get packages built for that new OS, so it's going to take a few days.  Stay tuned, I'll update this thread once the packages are published.

**Reply by madsi1m** [reply]
Cheers 👍

**Reply by nickdnk** [reply]
Hello @thresheek

Any update on this? Our PHP FPM alpine image is still failing to install nginx as it's now running 3.21.

(For stable though, not mainline, but same issue).

**Reply by thresheek** [reply]
Hi @nickdnk!

It's waiting on the following patchset to be reviewed & merged: https://github.com/nginx/pkg-oss/pull/23

**Reply by nickdnk** [reply]
Hello again

I'm kind of new to installing nginx on the FPM image docker image (on alpine), so: Is this kind of delay for most-recent versions of alpine a common occurrence or is this a special/rare situation? (Aside from it happening during Christmas, of course.)

**Reply by madsi1m** [reply]
> Hello again
> 
> I'm kind of new to installing nginx on the FPM image docker image (on alpine), so: Is this kind of delay for most-recent versions of alpine a common occurrence or is this a special/rare situation? (Aside from it happening during Christmas, of course.)

For now (and for testing only) I've pointed my 3.21 alpine container to use the nginx 3.20 repo and it seems to work fine

**Reply by nickdnk** [reply]
> > Hello again
> 
> > 
> 
> > I'm kind of new to installing nginx on the FPM image docker image (on alpine), so: Is this kind of delay for most-recent versions of alpine a common occurrence or is this a special/rare situation? (Aside from it happening during Christmas, of course.)
> 
> 
> 
> For now (and for testing only) I've pointed my 3.21 alpine container to use the nginx 3.20 repo and it seems to work fine

Alright, maybe that will work for me as well. I'll try it out.

**Reply by thresheek** [reply]
It is a rare situtation, typically we're much faster with that.  I think it's going to arrive today or tomorrow.

**Comment by thresheek** [comment]
...And the packages for Alpine 3.21 are now published.

Thanks for your pings.
